Microsoft.Toolkit.Win32/Microsoft.Toolkit.Win32.UI.Controls.WPF/WebView/Compatible/WebViewCompatibilityAdapter.cs

```diff
@@ -0,0 +1,106 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Windows;
+using Microsoft.Toolkit.Win32.UI.Controls.Interop.WinRT;
+
+namespace Microsoft.Toolkit.Win32.UI.Controls.WPF
+{
+    internal sealed class WebViewCompatibilityAdapter : WebBaseCompatibilityAdapter
+    {
+        private WebView _webView = new WebView();
```

Comment:
The WebView is loaded asynchronously on a dispatcher thread. Most properties that need state will need to wait (or be blocked and never return, PR to fix that already in progress). You'll need to hook the `Loaded` event to know when the dispatcher has begun initialization and you may call methods/properties that require state.